Common/Orders/LimitOrder.cs

```diff
@@ -37,6 +36,50 @@ public override decimal Value
         }
 
         /// <summary>
+        /// Create update request for pending orders. Null values will be ignored.
+        /// </summary>
+        public UpdateOrderRequest UpdateRequest(int? quantity = null, decimal? limitPrice = null, string tag = null)
+        {
+            return new UpdateOrderRequest
+            {
+                Id = Guid.NewGuid(),
+                OrderId = Id,
+                Created = DateTime.Now,
+                Quantity = quantity ?? Quantity,
+                LimitPrice = limitPrice ?? LimitPrice,
+                Tag = tag ?? Tag
+            };
+        }
+
+        /// <summary>
+        /// Apply changes after the update request is processed.
+        /// </summary>
+        public override void ApplyUpdate(UpdateOrderRequest request)
+        {
+            base.ApplyUpdate(request);
+
+            LimitPrice = request.LimitPrice;
+        }
+
+        /// <summary>
+        /// Create submit request.
+        /// </summary>
+        public static SubmitOrderRequest SubmitRequest(string symbol, int quantity, decimal limitPrice, string tag, SecurityType securityType, decimal price = 0, DateTime? time = null)
```

Comment:
Several of these `SubmitRequest` functions accept an optional `price` parameter but it never gets used. Let's remove it if it's unneeded.